www/grpc/proto/transaction.proto

```diff
@@ -167,8 +167,8 @@ message GetRawWithdrawTransactionRequest {
   string memo = 6;
 }
 
-// Request message for retrieving raw details of a transfer transaction.
-message RawTransfer {
+// Payload message for retrieving raw details of a transfer transaction.
+message TransferPayload {
```

Comment:
Why do we have two payload here: `TransferPayload` and `PayloadTransfer`? Why can't we use the old one?

Reply:
Because have duplicate memo and lock time fields.

Reply:
Here is two structs:

```proto
// Payload message for retrieving raw details of a transfer transaction.
message TransferPayload {
  // The sender's account address.
  string sender = 1;
  // The receiver's account address.
  string receiver = 2;
  // The amount to be transferred, specified in NanoPAC. Must be greater than 0.
  int64 amount = 3;
  // The transaction fee in NanoPAC. If not set, it is set to the estimated fee.
  int64 fee = 4;
}

// Payload for a transfer transaction.
message PayloadTransfer {
  // The sender's address.
  string sender = 1;
  // The receiver's address.
  string receiver = 2;
  // The amount to be transferred in NanoPAC.
  int64 amount = 3;
}
```

The `TransferPayload` uses `TransferPayload`. The `TransferPayload` should NOT include `fee`.
if you remove the `fee` both struct will be the same.